docs/en/ingest-management/elastic-agent/running-on-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,169 @@
+[[running-on-gke-standard-managed-by-fleet]]
+= Run {agent} on GKE standard managed by {fleet}
+
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+TIP: Running {ecloud} on Kubernetes? Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run {elastic-agent} on ECK].
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+[discrete]
+== Important Notes:
+
+On managed Kubernetes solutions like GKE, {agent} has no access several data sources. Find below the list of
+the non available data:
+
+1. Metrics from https://kubernetes.io/docs/concepts/overview/components/#control-plane-components[Kubernetes control plane]
+components are not available. Consequently metrics are not available for `kube-scheduler` and `kube-controller-manager`.
```

Comment:
```suggestion
components are not available. Consequently metrics are not available for `kube-scheduler` and `kube-controller-manager` datasets.
```
?

Reply:
Here I refer to the services not the data_streams.